cookbooks/cdo-varnish/test/shared/shared.rb

```diff
@@ -5,6 +5,7 @@
 #
 # $cloudfront == `true` for the CloudFront configuration.
 $cloudfront ||= false
```

Comment:
Does this have to be a global variable? Could it be provided as a parameter to the MockServer instead?